### DISCUSSION TITLE
divider in the bar_trailing not displayed

### DISCUSSION THREAD

**Discussion by BrentHuang** [flet-dev/flet#3171] [Q&A]
**Description**

divider in the bar_trailing not displayed

**Code example to reproduce the issue:**

```python
import logging
import flet as ft


def main(page):
    searchbar = ft.SearchBar(
        bar_leading=ft.Icon(ft.icons.SEARCH),
        bar_trailing=[ft.IconButton(ft.icons.CAMERA),
                      ft.VerticalDivider(),
                      ft.IconButton(ft.icons.SCANNER)])

    page.add(searchbar)


ft.app(target=main)
```

**Describe the results you received:**

![image](https://github.com/flet-dev/flet/assets/2862064/27bb3b7d-7e33-4930-a7b9-a8a64e7b0d99)

**Describe the results you expected:**

![image](https://github.com/flet-dev/flet/assets/2862064/cb1aec2e-5984-4ce8-8369-62a1a51c69ff)

**Additional information you deem important (e.g. issue happens only occasionally):**

**Flet version (`pip show flet`)**:

```
PS D:\workspace\code\plan> pip show flet      
Name: flet
Version: 0.22.0
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page:
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: d:\workspace\code\plan\.venv\Lib\site-packages
Requires: cookiecutter, fastapi, flet-runtime, packaging, qrcode, uvicorn, watchdog
Required-by:
```

**Operating system**:

版本	Windows 10 家庭中文版
版本号	22H2
安装日期	‎2023/‎9/‎14
操作系统内部版本	19045.4291
体验	Windows Feature Experience Pack 1000.19056.1000.0

**Additional environment details:**

flutter_windows_3.19.6-stable

**Comment by dickermoshe** [comment]
Try wrapping the divider in a box that limits it's height

**Reply by BrentHuang** [reply]
How to do that? Would you please show me the code?

**Comment by Bbalduzz** [comment]
Limit the divider height.
```
import logging
import flet as ft


def main(page):
    searchbar = ft.SearchBar(
        bar_leading=ft.Icon(ft.icons.SEARCH),
        bar_trailing=[
            ft.IconButton(ft.icons.CAMERA),
            ft.Container(ft.VerticalDivider(), height=30),
            ft.IconButton(ft.icons.SCANNER),
        ],
    )

    page.add(searchbar)


ft.app(target=main)

```

![image](https://github.com/flet-dev/flet/assets/81587335/fcbf6bb2-4216-4de0-804f-a3556e96ba49)

**Comment by dickermoshe** [comment]
What's interesting here is that most of these issues aren't flet specific and happen with flutter itself too.
These errors should be shown in terminal

**Comment by BrentHuang** [comment]
thank you!
